cli/integration/tests/waiter/test_cli.py

```diff
@@ -688,22 +688,47 @@ def test_update_does_patch(self):
             util.delete_token(self.waiter_url, token_name)
 
     def test_create_token_json(self):
+        create_fields = {'cpus': 0.1, 'mem': 128}
+
+        # Test with json from stdin
```

Comment:
Nitpick: these should be separate tests

Reply:
My thought was that the tests are testing the same thing, just pulling the JSON in from a different place, which is why they're nice to have together